<ui version="4.0" >
 <class>UniUpdate</class>
 <widget class="QDialog" name="UniUpdate" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>UniUpdate service</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../resources.qrc" >:/images/uniupdate.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QLabel" name="statusLabel" >
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="textVisible" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4" >
    <widget class="QFrame" name="changelogFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTextBrowser" name="changelogOutput" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="source" >
         <url>
          <string/>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QPushButton" name="updateButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>&amp;Update</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="changelogButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>&amp;Toggle changelog</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>250</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3" >
    <widget class="QPushButton" name="closeButton" >
     <property name="text" >
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>changelogButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>changelogFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>284</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel" >
     <x>31</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>UniUpdate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>545</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel" >
     <x>464</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
